eng/versioning/version_client.txt

```diff
@@ -96,36 +96,36 @@ com.azure.resourcemanager:azure-resourcemanager-sql;2.0.0-beta.4;2.0.0-beta.5
 com.azure.resourcemanager:azure-resourcemanager-storage;2.0.0-beta.4;2.0.0
 com.azure.resourcemanager:azure-resourcemanager-trafficmanager;2.0.0-beta.4;2.0.0-beta.5
 com.azure.resourcemanager:azure-resourcemanager-test;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-boot-starter-active-directory-b2c;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-spring-boot-starter-active-directory;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-spring-boot-starter-cosmos;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-spring-boot-starter-data-gremlin;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-spring-boot-starter-keyvault-secrets;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-spring-boot-starter-metrics;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-spring-boot-starter-servicebus-jms;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-spring-boot-starter;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-spring-boot;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-spring-cloud-autoconfigure;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-context;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-eventhubs-stream-binder;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-messaging;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-servicebus-queue-stream-binder;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-servicebus-stream-binder-core;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-servicebus-topic-stream-binder;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-starter-cache;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-starter-eventhubs-kafka;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-starter-eventhubs;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-starter-servicebus;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-starter-storage-queue;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-storage;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-stream-binder-test;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-cloud-telemetry;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-integration-core;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-integration-eventhubs;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-integration-servicebus;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-integration-storage-queue;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-integration-test;2.0.0-beta.1;2.0.0-beta.1
-com.azure:azure-spring-starter-storage;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-boot-starter-active-directory-b2c;3.0.0-beta.1;3.0.0-beta.1
```

Comment:
We should place all spring modules together and order them first by domain area and then alphabetically.